src/test/java/org/prebid/server/bidder/ttx/TtxBidderTest.java

```diff
@@ -129,6 +136,120 @@ public void makeHttpRequestsShouldChangeOnlyFirstImpExt() {
                         mapper.valueToTree(ExtPrebid.of(null, ExtImpTtx.of("11", null, "3"))));
     }
 
+    @Test
+    public void makeHttpRequestsShouldReturnErrorIfAnyOfVideoParamsNotPresent() {
```

Comment:
If you want to test Any, you will have several tests for specific case, so remove `Any`